iac-extensions/arm/src/test/resources/checks/HardCodedCredentialsCheck/HardCodedCredentialsCheck.json

```diff
@@ -0,0 +1,34 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+  },
+  "resources": [
+    {
+      "type": "Microsoft.Sql/servers",
+      "apiVersion": "2023-05-01-preview",
+      "name": "noncompliant",
+      "properties": {
+        "administratorLogin": "admin",
+        "administratorLoginPassword": "password",
+        "password": "password",
+        "secret":  "secret",
+        "adminPassword":  "password",
+        "adminUsername": "admin"
+      }
+    },
+    {
+      "type": "Microsoft.Sql/servers",
+      "apiVersion": "2023-05-01-preview",
+      "name": "compliant",
+      "properties": {
+        "administratorLogin": "[parameters('administratorLogin')]",
+        "administratorLoginPassword": "[parameters('administratorLoginPassword')]",
+        "password": "[parameters('password')]",
+        "secret":  "[parameters('secret')]",
+        "adminPassword":  "[parameters('adminPassword')]",
+        "adminUsername": ""
+      }
+    }
```

Comment:
What do you think about adding real examples here (and in bicep)?
The `Microsoft.Sql/servers` uses only `administratorLogin` & `administratorLoginPassword` properties, but not others.